CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 cmake_minimum_required(VERSION 3.23)
+cmake_policy(SET CMP0144 NEW)
```

Comment:
Why did we add this policy?

Reply:
`CMP0144` detects library folders in allcaps rather than with case sensitivity. There were warnings about this until I explicitly specified it.